content/release-10-17-0/platform-10-17-0.md

```diff
@@ -137,6 +137,13 @@ Fix Version</th>
 <td>10.16.13.0</td>
 </tr>
 
+<tr>
+<td>Authentication</td>
+<td>Fixed an issue with the device requests counter being increased while switching between the standard applications (Administration, Cockpit, Device management).</td>

```

Comment:
see 10.16 PR - i suggested removing the "s" at the end of "requests" - but not sure how the component is actually called